nextcord/user.py

```diff
@@ -159,7 +160,11 @@ def avatar(self) -> Optional[Asset]:
 
     @property
     def default_avatar(self) -> Asset:
-        """:class:`Asset`: Returns the default avatar for a given user. This is calculated by the user's discriminator."""
+        """
+        :class:`Asset`: Returns the default avatar for a given user.
+
+        This is calculated by the user's discriminator.
+        """
```

Comment:
Starting on the first line would be more consistent with the other docstrings

```suggestion
        """:class:`Asset`: Returns the default avatar for a given user.
        
        This is calculated by the user's discriminator.
        """
```